src/mbgl/util/mapbox.cpp

```diff
@@ -85,6 +95,11 @@ std::string normalizeGlyphsURL(const std::string& url, const std::string& access
     }
 
     std::vector<std::string> pathname = getMapboxURLPathname(url);
+
+    if (pathname.size() < 3) {
```

Comment:
This one should be `pathname.size() < 4`, since `pathname[3]` is used unconditionally.